Server Time Zone Unrecognized

```
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value 'PDT' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
```

Seems to be an interaction between JDBC, Java8, and Mariadb.  Easy fix (i.e. not mucking about server-side) involves specifying the time zone in the connection URL `serverTimezone=UTC` (note extreme case sensitivity)

--- a/baseConfig/cromServer.sh
+++ b/baseConfig/cromServer.sh
@@ -18,6 +18,17 @@ source ${1}
 # Load the Cromwell Module
 module load cromwell/49-Java-1.8
 
+# All this to make it a little more readable.  Put JDBC connection
+# options in a bash array
+jdbc_options=(\
+  rewriteBatchedStatements=true \
+  serverTimezone=UTC \
+)
+
+# Then encode for URL (ampersands and all that)
+# see here: https://stackoverflow.com/questions/1527049/how-can-i-join-elements-of-an-array-in-bash
+jdbc_connect_params=$(IFS=\& ; echo "${jdbc_options[*]}")
+
 # Run your server!
 java -Xms4g \
     -Dconfig.file=${CROMWELLCONFIG} \
@@ -26,7 +37,7 @@ java -Xms4g \
     -Dbackend.providers.gizmo.config.root=${SCRATCHPATH} \
     -Dworkflow-options.workflow-log-dir=${WORKFLOWLOGDIR} \
     -Dworkflow-options.final_workflow_outputs_dir=${WORKFLOWOUTPUTSDIR} \
-    -Ddatabase.db.url=jdbc:mysql://mydb:${CROMWELLDBPORT}/${CROMWELLDBNAME}?rewriteBatchedStatements=true\&serverTimezone=UTC \
+    -Ddatabase.db.url=jdbc:mysql://mydb:${CROMWELLDBPORT}/${CROMWELLDBNAME}?${jdbc_connect_params} \
     -Ddatabase.db.user=${CROMWELLDBUSERNAME} \
     -Ddatabase.db.password=${CROMWELLDBPASSWORD} \
     -Dwebservice.port=${2} \

--- a/baseConfig/cromServer.sh
+++ b/baseConfig/cromServer.sh
@@ -18,7 +18,6 @@ source ${1}
 # Load the Cromwell Module
 module load cromwell/49-Java-1.8
 
-
 # Run your server!
 java -Xms4g \
     -Dconfig.file=${CROMWELLCONFIG} \
@@ -27,7 +26,7 @@ java -Xms4g \
     -Dbackend.providers.gizmo.config.root=${SCRATCHPATH} \
     -Dworkflow-options.workflow-log-dir=${WORKFLOWLOGDIR} \
     -Dworkflow-options.final_workflow_outputs_dir=${WORKFLOWOUTPUTSDIR} \
-    -Ddatabase.db.url=jdbc:mysql://mydb:${CROMWELLDBPORT}/${CROMWELLDBNAME}?rewriteBatchedStatements=true \
+    -Ddatabase.db.url=jdbc:mysql://mydb:${CROMWELLDBPORT}/${CROMWELLDBNAME}?rewriteBatchedStatements=true\&serverTimezone=UTC \
     -Ddatabase.db.user=${CROMWELLDBUSERNAME} \
     -Ddatabase.db.password=${CROMWELLDBPASSWORD} \
     -Dwebservice.port=${2} \

